test/org/javarosa/core/model/instance/test/SimpleBenchmarkTests.java

```diff
@@ -0,0 +1,104 @@
+package org.javarosa.core.model.instance.test;
+
+
+
+import org.javarosa.core.util.PathConst;
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.instance.ExternalDataInstance;
+import org.javarosa.core.reference.InvalidReferenceException;
+import org.javarosa.core.reference.ReferenceManager;
+import org.javarosa.xform.parse.FormParserHelper;
+import org.javarosa.xml.util.InvalidStructureException;
+import org.javarosa.xml.util.UnfullfilledRequirementsException;
+import org.junit.Test;
+import org.xmlpull.v1.XmlPullParserException;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Collections;
+
+import static org.javarosa.core.reference.ReferenceManagerTestUtils.buildReferenceFactory;
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public class SimpleBenchmarkTests {
```

Comment:
Avoid `PathConst` and replace with calls to the `r(String filename)` method.

Also, watch out for the code formatting: some indentations and whitespaces don't follow the code style from the rest of the file.